### PR TITLE
new projects from VSP and PEARC20 trainings

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

**Comment by edquist** [comment]
Hi @lmichael107,

>  New Organization 'Southwest Oklahoma State University' requires OSG approval

Any chance this is the same as one of these?

```
projects/SO10GU.yaml:7:            Organization: Oklahoma State University
projects/TG-CHE150012.yaml:13:     Organization: Oklahoma State University
projects/atlas.org.okstate.yaml:6: Organization: Oklahoma State University
projects/atlas.org.ou.yaml:6:      Organization: Oklahoma University
projects/nicesims.yaml:14:         Organization: University of Oklahoma
projects/polymer.yaml:8:           Organization: University of Oklahoma
```

**Comment by lmichael107** [comment]
This is very much a different university than any of those.

File: projects/UWMadison_Wright.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-Description: Project for Daniel Wright hydroclimate extremes research group from UW-Madison
+Description: Project for Daniel Wright's research group at UW-Madison
 Department: Civil and Environmental Engineering
-FieldOfScience: Atmospheric Sciences
+FieldOfScience: Civil Engineering
 Organization: University of Wisconsin-Madison
 PIName: Daniel Wright
 
-ID: '698'
+ID: '705'
```
Comment (brianhlin) [inline projects/UWMadison_Wright.yaml:8]:
```suggestion
ID: '698'
```

Revert the change in project ID
